lib/components/DropdownMenu/DropdownMenu.tsx

```diff
@@ -1,72 +1,98 @@
 import { useId } from 'react';
-import classNames from 'classnames';
 
-export interface DropdownMenuOptionsData<T extends string = string> {
-  value: T;
+export interface DropdownMenuSelectProps<T extends DropdownMenuOptionsData> {
```

Comment:
Not sure if this interface need to be exported, but if it is useful to be exported then it could benefit for adding the JSDoc for the props.

Reply:
I add them.